projects/core/directives/dropdown/dropdown-manual.directive.ts

```diff
@@ -7,13 +7,12 @@ import {TuiDropdownDriver} from './dropdown.driver';
     selector: '[tuiDropdownManual]',
     providers: [TuiDropdownDriver, tuiAsDriver(TuiDropdownDriver)],
 })
-export class TuiDropdownManual implements OnChanges {
+export class TuiDropdownManual {
     private readonly driver = inject(TuiDropdownDriver);
 
-    @Input()
-    public tuiDropdownManual: boolean | '' = false;
+    public readonly tuiDropdownManual = input<boolean | ''>(false);
 
-    public ngOnChanges(): void {
-        this.driver.next(!!this.tuiDropdownManual);
-    }
+    protected readonly driverEffect = effect(() =>
+        this.driver.next(!!this.tuiDropdownManual()),
+    );
```

Comment:
Are we ok switching all the ngOnChanges to effects? There are some timing differences between those, I'm a bit worried this might cause issues somewhere.